File: python_preparation/pattern_printing/star/pyramid.py
def print_pyramid(rows):
    for i in range(rows):
        print(" " *(rows-i)+ "*" * (2 * i+1))
        
        
# Example usage:
rows = 5
print_pyramid(rows)
def diamond_ptrn(n):
    for i in range(n):
        for j in range(0,n-i-1):
            print(end=' ')
        for j in range(0,i+1):
            print('*', end=' ')
        print()


print()


n = 5
for i in range(0,n):
    print(' '*(n-i-1)+'*'*(2*i+1))

diamond_ptrn(5)

File: JAVA/pattern/star/pyramid.java
public  class pyramid{
    public static void main(String[] args) {
        int rows = 9;
        for(int i=0;i<rows;i++){
            for(int j=0;j<rows-i-1;j++){
                System.out.print(" ");
            }

            for(int k=0;k<=i;k++){
                System.out.print("* ");
            }
            System.out.println();
        }
    }
}